src/components/pages/AdminHome/AdminCourses.js

```diff
@@ -1,11 +1,80 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import AdminSidebar from './AdminSidebar';
 import '../../../styles/index.less';
+import { connect } from 'react-redux';
+import CourseCard from './AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/coursesActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { Layout, Input } from 'antd';
+const { Search } = Input;
+// leaving this import in to allow for the modal coming from add-course
+// import AdminAddCourses from '../AdminAddCourse';
+
+// need to upgrade the courses router/modal to get the instructor name (2 seconds update in modal)
+
+function AdminCourses(props) {
+  const { courses } = props;
+  const dispatch = useDispatch();
+  const idToken = useOktaAuth().oktaAuth.getIdToken();
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
+  }, [dispatch, idToken]);
+
+  const history = useHistory();
+
+  const handleClick = () => {
+    history.push('/admin-add-course');
+  };
+  const onSearch = searchTerm => {
+    console.log(searchTerm);
+    courses.find(item => item === searchTerm);
+  };
+
+  const { Content } = Layout;
 
-export default function AdminCourses() {
   return (
-    <div className="admin-courses-container">
+    <Layout>
       <AdminSidebar />
-    </div>
+      <Content>
+        <div class="table-container">
+          <div class="header-container">
+            <h1>Courses</h1>
+          </div>
+          <div className="courses-page-flex">
+            <div className="courses-left">
+              {/* placeholder for now - holding for universal component */}
```

Comment:
why do we need this placeholder?

Reply:
So lisa knows where to put the search bar component she's building